<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="2" column="3">
    <widget class="QLineEdit" name="lineedit_number">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>shape</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="combobox_shape"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>volume</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="combobox_volume"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>distribution</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="combobox_distribution">
     <item>
      <property name="text">
       <string>random</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cubic</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="checkbox_union">
     <property name="text">
      <string>union</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushbutton_close">
        <property name="text">
         <string>close</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushbutton_apply">
        <property name="text">
         <string>apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_numberspacing">
     <property name="text">
      <string>number</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="checkbox_rotation">
     <property name="text">
      <string>random rotation</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>combobox_shape</tabstop>
  <tabstop>combobox_volume</tabstop>
  <tabstop>combobox_distribution</tabstop>
  <tabstop>lineedit_number</tabstop>
  <tabstop>checkbox_union</tabstop>
  <tabstop>checkbox_rotation</tabstop>
  <tabstop>pushbutton_close</tabstop>
  <tabstop>pushbutton_apply</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
